joint_limits_interface/test/joint_limits_interface_test.cpp

```diff
@@ -376,28 +376,41 @@ TEST_F(VelocityJointSaturationHandleTest, EnforceAccelerationBounds)
 
   pos = 0.0;
   double cmd;
+  ros::Duration long_enough(1000.0);
```

Comment:
Should be const, and should include comment of why 1000sec was selected, even if it's just `// An arbitrarily long time, sufficient to suppress acceleration limits`,

Reply:
awesome